examples/3rdparty/python/BUILD

```diff
@@ -4,6 +4,36 @@
 # see/edit requirements.txt in this directory to change deps.
 python_requirements()
 
+# NB: google-cloud-bigquery has the following prelude upon import:
+#
+# from pkg_resources import get_distribution
+# __version__ == get_distribution("google-cloud-bigquery").version
+#
+# which will fail at import time with pkg_resources.DistributionNotFound.
+# The argument `prepend_to_pythonpath=True` will work around this particular error by ensuring the
+# requirements are within a pex file located at the at the front of the PEX_PATH when executing
+# pytest. Note that even though this works, it will still show the following warning:
+#
+# pex/environment.py:330: PEXWarning: The `pkg_resources` package was loaded from a pex vendored
+# version when declaring namespace packages defined by google-cloud-bigquery 1.10.0. The
+# google-cloud-bigquery 1.10.0 distribution should fix its `install_requires` to include
+# `setuptools`.
+python_requirement_library(
+  name='google-cloud-bigquery',
+  requirements=[
+    python_requirement('google-cloud-bigquery==1.10.0'),
+  ],
+  # TODO: The `prepend_to_pythonpath` hack seems to invalidate some assumptions that pants makes in
+  # creating the pytest pex. In this case, it seems to require the `requests` module early at import
+  # time, as was observed in the stack trace. If an error appears at module import time like this,
```

Comment:
As observed in which stacktrace? Could gist it for inclusion.

Reply:
I spent a while trying to repro this, but I think it was because of a stale cached pytest pex being used that I thought this was a problem. In running it now it doesn't appear the `requests` dependency is necessary, so I'm probably going to just remove this.